### PR TITLE
3.11.188 release notes stub and latest tags

### PR DESCRIPTION
**Pull request by sheriff-rh**
Ships 2020-03-18.

### CODE REVIEW

**Comment by sheriff-rh** [comment]
@ahardin-rh  and @codyhoag  can you review for correctness? 9 RHSAs means a lot of anchors/URLs/H3s that need to match. i triple checked before committing, but extra eyes are always helpful 😝

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -3,10 +3,9 @@
 {product-author}
 {product-version}
 ifdef::openshift-enterprise[]
-:latest-tag: v3.11.170
-endif::[]
```
Comment (codyhoag) [inline install_config/aggregate_logging.adoc]: Was removing this first `endif::[]` intentional?

Comment (sheriff-rh) [reply install_config/aggregate_logging.adoc]: whoa! not sure how that happened! thanks, fixing now.

**Comment by codyhoag** [comment]
Shew! Lots of RHSAs! :sweat_smile: Other than the endif question, LGTM!

**Pull request merged**